com.unity.render-pipelines.core/Runtime/Utilities/CoreRenderPipelinePreferences.cs

```diff
@@ -10,80 +15,60 @@ namespace UnityEngine.Rendering
+        static readonly ConcurrentStack<object> s_ColorPref = new ConcurrentStack<object>();
+
+        #region Volumes Gizmo Color
 
-        // Added default Colors so that they can be reverted back to these values
         static Color s_VolumeGizmoColorDefault = new Color(0.2f, 0.8f, 0.1f, 0.5f);
-        static Color s_VolumeGizmoColor = s_VolumeGizmoColorDefault;
-        static readonly Color kPreviewCameraBackgroundColorDefault = new Color(82f / 255.0f, 82f / 255.0f, 82.0f / 255.0f, 0.0f);
+        private static Func<Color> GetColorPrefVolumeGizmoColor;
 
-        public static Color volumeGizmoColor
-        {
-            get => s_VolumeGizmoColor;
-            set
-            {
-                if (s_VolumeGizmoColor == value) return;
-                s_VolumeGizmoColor = value;
-                EditorPrefs.SetInt(Keys.volumeGizmoColor, (int)ColorUtils.ToHex(value));
-            }
-        }
+        public static Color volumeGizmoColor => GetColorPrefVolumeGizmoColor();
 
-        public static Color previewBackgroundColor => kPreviewCameraBackgroundColorDefault;
+        #endregion
 
-        static class Keys
-        {
-            internal const string volumeGizmoColor = "CoreRP.Volume.GizmoColor";
-        }
+        #region Preview Camera Background Color
 
-        [SettingsProvider]
-        static SettingsProvider PreferenceGUI()
-        {
-            return new SettingsProvider("Preferences/Colors/SRP", SettingsScope.User)
-            {
-                guiHandler = searchContext =>
-                {
-                    if (!m_Loaded)
-                        Load();
-
-                    Rect r = EditorGUILayout.GetControlRect();
-                    r.xMin = 10;
-                    EditorGUIUtility.labelWidth = 251;
-                    volumeGizmoColor = EditorGUI.ColorField(r, "Volume Gizmo Color", volumeGizmoColor);
-
-                    if (GUILayout.Button(Styles.userDefaults, GUILayout.Width(120)))
-                    {
-                        RevertColors();
-                    }
-                }
-            };
-        }
+        static readonly Color kPreviewCameraBackgroundColorDefault = new Color(82f / 255.0f, 82f / 255.0f, 82.0f / 255.0f, 0.0f);
+        public static Color previewBackgroundColor => kPreviewCameraBackgroundColorDefault;
 
-        static void RevertColors()
-        {
-            volumeGizmoColor = s_VolumeGizmoColorDefault;
-        }
+        #endregion
 
         static CoreRenderPipelinePreferences()
         {
-            Load();
+            GetColorPrefVolumeGizmoColor = RegisterPreferenceColor("Scene/Volume Gizmo", s_VolumeGizmoColorDefault);
         }
 
-        static void Load()
+        /// <summary>
+        /// This function provides a way of adding a <see cref="PrefColor"/> into the panel 'Preferences\Colors' />
+        /// </summary>
+        /// <param name="name">The name that the color will have on the panel, in form of 'group/name'</param>
```

Comment:
The name the color has in the **Colors** panel. This is in the format of 'group/name'.